src/components/asset-list/AssetListItemSkeleton.js

```diff
@@ -39,36 +39,33 @@ const FakeRow = styled(Row).attrs({
   justify: 'space-between',
   paddingBottom: 5,
   paddingTop: 5,
-})(Row);
```

Comment:
Why was `Row` passed here previously? Is that a mistake?

Reply:
Yes, it was a mistake